typescript/sdk/src/providers/MultiProvider.ts

```diff
@@ -413,7 +454,10 @@ export class MultiProvider<MetaExt = {}> extends ChainMetadataManager<MetaExt> {
     }
     const txReq = await this.prepareTx(chainNameOrId, tx);
     const signer = this.getSigner(chainNameOrId);
-    const response = await signer.sendTransaction(txReq);
+
+    const response = await signer.sendTransaction({
+      ...txReq,
+    });
```

Comment:
why is this necessary?

Reply:
Leftovers from when we had to hardcode gasLimit for zksync local node, will remove